src/plugins/lightbox/lightbox.js

```diff
@@ -167,10 +168,17 @@ var componentName = "wb-lbx",
 					this.contentContainer.attr( "data-pgtitle", document.getElementsByTagName( "H1" )[ 0 ].textContent );
 
 					trapTabbing( $wrap );
+
+					if ( $container.html().trim() !== "" ) {
+						var contentHtml = $container.html();
+						$container.empty().html( "<dialog class='mfp-content' open>" + contentHtml + "</dialog>" );
```

Comment:
Add a value for the open attribute, like `open="open"`

Reply:
thx. updated